chia/wallet/wallet_state_manager.py

```diff
@@ -383,11 +383,10 @@ async def create_more_puzzle_hashes(
                     )
                 self.log.info(f"Done: {creating_msg} Time: {time.time() - start_t} seconds")
             await self.puzzle_store.add_derivation_paths(derivation_paths)
-            await self.add_interested_puzzle_hashes(
```

Comment:
does `add_interested_puzzle_hashes()` also subscribe to the puzzle hashes?

Reply:
Yes and to also answer your other question below, i added the subscription call below in the `if len(derivation_paths) > 0:` path which then retains the same behaviour as before, see https://github.com/Chia-Network/chia-blockchain/blob/0673cb624cffb938b2b3a7f7bdc511ce3b31a5ea/chia/wallet/wallet_state_manager.py#L1731-L1732